x/twap/api_test.go

```diff
@@ -238,9 +237,158 @@ func (s *TestSuite) TestGetArithmeticTwap() {
 	for name, test := range tests {
 		s.Run(name, func() {
 			s.SetupTest()
-			for _, record := range test.recordsToSet {
-				s.twapkeeper.StoreNewRecord(s.Ctx, record)
+			s.preSetRecords(test.recordsToSet)
+			s.Ctx = s.Ctx.WithBlockTime(test.ctxTime)
+
+			var twap sdk.Dec
+			var err error
+
+			twap, err = s.twapkeeper.GetArithmeticTwap(s.Ctx, test.input.poolId,
+				test.input.quoteAssetDenom, test.input.baseAssetDenom,
+				test.input.startTime, test.input.endTime)
+
+			if test.expErrorStr != "" {
+				s.Require().Error(err)
+				s.Require().Contains(err.Error(), test.expErrorStr)
```

Comment:
Why do we care which error was returned?

Reply:
@ValarDragon here is a recent example https://github.com/osmosis-labs/osmosis/pull/2536#discussion_r958758612

Reply:
Fixed all errors I observed in the file. Generally, let's create separate issues for such things. I think it's best to keep the scope focused on the actual change instead of fixing drive-by problems